client/state/utils.js

```diff
@@ -37,14 +40,18 @@ export function isValidStateWithSchema( state, schema, debugInfo ) {
 			return obj;
 		}
 		warn(
-			'state validation failed\nfor state:',
+			'state validation failed for state:\n',
 			state,
-			'\nagainst schema:',
-			schema,
-			'\nwith reason:',
-			prettifyArgument( validate.errors ),
-			'\nsource:',
-			prettifyArgument( debugInfo ) || '(none given)'
```

Comment:
Let's keep `debugInfo`: it's currently the only way to get Chrome devtools to tell you which reducer this error originates from, even though it's not pretty.